package-lock.json

```diff
@@ -215,46 +215,6 @@
       "resolved": "https://registry.npmjs.org/@emotion/weak-memoize/-/weak-memoize-0.2.4.tgz",
       "integrity": "sha512-6PYY5DVdAY1ifaQW6XYTnOMihmBVT27elqSjEoodchsGjzYlEsTQMcEhSud99kVawatyTZRTiVkJ/c6lwbQ7nA=="
     },
-    "@icgc-argo/uikit": {
-      "version": "1.0.6",
-      "resolved": "https://registry.npmjs.org/@icgc-argo/uikit/-/uikit-1.0.6.tgz",
-      "integrity": "sha512-LENeuVDCyGFEX0uPXCD7JVVcu6LXaIa+BXwngQefoFnpTgPum03zFo/9A9bJp1+dggIIsgB4l4QZXKutTnN0mw==",
-      "requires": {
-        "@babel/runtime-corejs2": "^7.7.2",
-        "@emotion/core": "^10.0.10",
-        "@emotion/styled": "^10.0.11",
-        "@types/lodash": "^4.14.138",
-        "@types/react-table": "6.8.5",
-        "clsx": "^1.0.4",
-        "color": "^3.1.2",
-        "date-fns": "^1.30.1",
-        "element-resize-detector": "^1.1.15",
-        "emotion-theming": "^10.0.10",
-        "i": "^0.3.6",
-        "jsdom": "^14.0.0",
-        "lodash": "^4.17.14",
-        "prop-types": "^15.7.2",
-        "react-grid-system": "^4.4.6",
-        "react-table": "6.10.0",
-        "react-tippy": "^1.2.3",
-        "recompose": "^0.30.0"
-      },
-      "dependencies": {
-        "date-fns": {
-          "version": "1.30.1",
-          "resolved": "https://registry.npmjs.org/date-fns/-/date-fns-1.30.1.tgz",
-          "integrity": "sha512-hBSVCvSmWC+QypYObzwGOd9wqdDpOt+0wl0KbU+R+uuZBS1jN8VsD1ss3irQDknRj5NvxiTF6oj/nDRnN/UQNw=="
-        },
-        "react-table": {
-          "version": "6.10.0",
-          "resolved": "https://registry.npmjs.org/react-table/-/react-table-6.10.0.tgz",
-          "integrity": "sha512-s/mQLI1+mNvlae45MfAZyZ04YIT3jUzWJqx34s0tfwpDdgJkpeK6vyzwMUkKFCpGODBxpjBOekYZzcEmk+2FiQ==",
-          "requires": {
-            "classnames": "^2.2.5"
-          }
-        }
-      }
-    },
```

Comment:
what happened to uikit?

Reply:
@hlminh2000 good question. npm ci should fix?

Reply:
@hlminh2000 this package-lock isn't needed
generated from running npm install commands in the root instead of in `scripts` and `website`